src/main/resources/db/migration/V56__Add_missing_documents_record_tables.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE missing_documents_record (
+          id UUID PRIMARY KEY,
+          recall_id UUID NOT NULL REFERENCES recall (id),
+          email_id UUID NOT NULL REFERENCES document (id),
+          detail text NOT NULL,
+          version INTEGER NOT NULL,
+          created_by_user_id UUID NOT NULL,
+          created_date_time TIMESTAMPTZ NOT NULL,
+          UNIQUE (recall_id, version)
+);
+
+CREATE TABLE missing_document_category (
+    missing_document_record_id    UUID    NOT NULL REFERENCES missing_documents_record (id),
+    document_category   VARCHAR(32) NOT NULL,
```

Comment:
FYI the other store we have for the same info - I believe - is defined as `category  TEXT NOT NULL` so not as `VARCHAR`.
Of course there is a case for extracting those to a separate table too - but I don't think we need that for now.